### PR TITLE
Adding lln-sina recipe

### PR DESCRIPTION
**Pull request by doutriaux1**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by HaluskaR** [comment]
I’m okay with being listed as maintainer. Thanks!

**Comment by doutriaux1** [comment]
@conda-forge/staged-recipes this is ready to go. Thanks!

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:15]: Please remove the extra lines, the build scripts, and try something like:

```yaml
script:
  - cd python
  - {{ PYTHON }} -m pip install . -vv
```

here in build. That way we can have everything in a single file.

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:20]: Please add `pip` instead of setuptools. See the builds script above.

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:22]: Unless this module imports itself in setup.py you can remove siz and sqlalchemy here. Also, rename the `build` section to `host`.

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:31]: Simplify the test phase with an import test instead of a command test that calls an import and add a `pip check` test.

```yaml
test
  commands:
    - python -c "import sina"
```

```yaml
test:
  imports:
    - sina
  commands:
    - pip check
  requires:
    - pip
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]
Almost good to go. Some minor changes before we merge.

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:26]: We no longer build any Python <3.4 so I guess it is safe to remove this.

File: recipes/llnl-sina/meta.yaml
Comment (ocefpaf) [inline recipes/llnl-sina/meta.yaml:18]: We need to add the entrypoint here to fix Windows.

```yaml
  entry_points:
    - sina = sina.cli.driver:main
```

File: recipes/llnl-sina/meta.yaml
Comment (doutriaux1) [reply recipes/llnl-sina/meta.yaml:26]: @ocefpaf internally we still need to build for Python<3.4 do you mind if we keep this one in? I would like to use the same recipe everywhere. Thanks!

Comment (ocefpaf) [reply recipes/llnl-sina/meta.yaml:29]: Sure. That is not a problem.

**Pull request merged**
